### PR TITLE
Add DL3+ notebooks

### PR DESCRIPTION
**Pull request by chaimain**
Adding 5 notebooks:
- MC DL2 to IRF (source-independent, fixed global and energy-dependent gammaness and theta cuts) 
- DL2 to DL3 (source-independent, fixed global and energy-dependent gammaness cuts)
- DL3 to DL4 (using gammapy v0.19 and saving DL4 as OGIP files)
- plot LC from DL4 (OGIP files)
- plot SED from DL4 (OGIP files)

The energy-dependent cuts procedures follows #709

### CODE REVIEW

**Comment by rlopezcoto** [comment]
After v0.9 we should try to make all notebooks work with the current version of lstchain

File: lstchain/scripts/lstchain_merge_run_summaries.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
+from pathlib import Path
```
Comment (maxnoe) [inline lstchain/scripts/lstchain_merge_run_summaries.py:1]: This is a script from another PR and does not belong here

**Review by maxnoe — CHANGES_REQUESTED** [review]
Why are there notebooks for things that are supposed to be done with the tools, e.g. `create_dl3_events_file` and `create_irfs`?

**Comment by chaimain** [comment]
> Why are there notebooks for things that are supposed to be done with the tools, e.g. `create_dl3_events_file` and `create_irfs`?

To just show the workflow of the Tools.

**Comment by maxnoe** [comment]
@chaimain But the notebooks duplicate code from the tools.

**Comment by chaimain** [comment]
> @chaimain But the notebooks duplicate code from the tools.
Yes. I do not understand, how I can explain the workflow of the Tools, without doing so in a jupyter notebook. Can you elaborate more on this?

**Comment by maxnoe** [comment]
@chaimain You show how you run the tools and explain what they do shortly in prose? Have a look in the inputs/outputs?

We really should not duplicate the code in notebooks that is supposed to be run via command line tools.

People who really want to understand the code need to look at the code, we shouldn't repeat it in a notebook

**Comment by chaimain** [comment]
> @chaimain You show how you run the tools and explain what they do shortly in prose? Have a look in the inputs/outputs?
> 
> We really should not duplicate the code in notebooks that is supposed to be run via command line tools.
> 
> People who really want to understand the code need to look at the code, we shouldn't repeat it in a notebook


Ok. I understand it now. Thanks!
I will fix the notebooks accordingly.

**Comment by chaimain** [comment]
@maxnoe should the `--help-all` section be run in the notebook?

**Comment by rlopezcoto** [comment]
as commented at the beginning of the discussion, after the new release we should update all notebooks to make them work with the current version of lstchain and some of them like these ones should not contain the information that is better explained elsewhere (as in gammapy notebooks)

**Review by rlopezcoto — APPROVED** [review]

**Pull request merged**